_includes/advantages/advantages.scss

```diff
@@ -51,11 +51,20 @@
     .advantage {
       display: grid;
       grid-template-columns: 66px 294px;
```

Comment:
Do we need to hardcode width of second column at all? 
Why isn't it just 1fr? It should take the rest of the space in the row.
And you will not need so many media-queries to update that hardcoded width